nbextensions/appCell2/widgets/appCellWidget.js

```diff
@@ -83,7 +81,8 @@ define([
         p = t('p'),
         blockquote = t('blockquote'),
         appStates = AppStates,
-        { toBoolean } = utils;
+        { toBoolean } = utils,
+        developerMode = false;
```

Comment:
Setting `developerMode` to true will turn on the display of the FSM bar, which shows the FSM state and the current job ID.